lib/install/validate/validator.go

```diff
@@ -936,6 +940,23 @@ func (v *Validator) syslog(op trace.Operation, conf *config.VirtualContainerHost
 	}
 }
 
+func (v *Validator) kubelet(op trace.Operation, conf *config.VirtualContainerHostConfigSpec, input *data.Data) {
+	defer trace.End(trace.Begin("", op))
+
+	if input.Kubelet.ServerURL == nil || input.Kubelet.ConfigFile == nil {
+		return
+	}
+
+	conf.KubernetesServiceURL = *input.Kubelet.ServerURL
+	conf.KubeletConfigFile = *input.Kubelet.ConfigFile
+
+	err := kubelet.ReadKubeletConfigFile(op, conf)
+	if err != nil {
+		v.NoteIssue(fmt.Errorf("failed to load Kubelet config file"))
+		op.Logger.Error("Failed to load Kubelet config file: %s", err.Error())
```

Comment:
It doesn't seem like the other methods in this file explicitly log when noting an error. Instead, they rely on the logic which prints all errors at the end of the validation process. Is there a reason not to follow that pattern here?